dataland-keycloak/dataland_theme/login/templates/messages/messages_en.properties

```diff
@@ -11,3 +11,17 @@ confirmEmailAddressVerification=Confirm validity of your e-mail address by click
 emailVerifiedMessageHeader=Account verified
 emailVerifiedMessage=Your email address has been verified.
 emailVerifyInstruction2=Haven''t received a verification link in your email?
+
+federatedIdentityConfirmLinkMessage=A user with {0} {1} already exists on Dataland. Do you want to link this new login method to your existing account?
+emailLinkIdpTitle=Link your {0} account
+emailLinkIdp1=An email with instructions to link your {0} account {1} with your Dataland account has been sent to you.
```

Comment:
Da hier eine neue Email versand wird würde ich vorschlagen das auch in die manual maintenance mit aufzunehmen (da alle anderen Emails dort auch getestet werden)

Reply:
Ich habe zwei einträge fürs prüfen von account linking (via Email und via login) eingefügt